src/components/AxisCollection/YAxis.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import * as d3 from 'd3';
+import PropTypes from 'prop-types';
+import { createYScale } from '../../utils/scale-helpers';
+
+export default class YAxis extends Component {
+  static propTypes = {
```

Comment:
Let's stick with same approach about declaring propTypes and defaultProps either inside classes as statics or outside. Personally I prefer to put them outside but it's ok to have them as statics but consistently throughout the code.

Reply:
Consistency is smart. Cleanup is yet to be done.

Reply:
I just happy to stick with [AirBnb style guide](https://github.com/airbnb/javascript/tree/master/react#ordering). Nothing else